### DISCUSSION TITLE
How to use JSON list to PlutoRows in flutter

### DISCUSSION THREAD

**Discussion by rcvamsivardhan** [bosskmk/pluto_grid#432] [Q&A]
Hi Everyone,

I would like to request you to provide any snippet to convert API result which is JSON, I want to use JSON result set to PlutoRows.

I didn't get any clue to do the same.

Kindly guide me.

Thanks & Regards
Vamsi

**Comment by bosskmk** [comment]
@rcvamsivardhan 
Please refer to the link below.
https://github.com/bosskmk/pluto_grid/issues/284#issuecomment-1003894598
